### ISSUE TITLE
Support composite types

### ISSUE THREAD

**Issue by BugGambit** [oguimbal/pgsql-ast-parser#54]
Great project! 🙏 
FYI
Version `8.1.1` isn't able to parse a create a statement for composite types:
```
CREATE TYPE weight AS (
  unit text,
  value double precision
);
```

Parsing the statement will resolve in:
```
Syntax error at line 2 col 25:

        CREATE TYPE weight AS (
                              ^
    Unexpected lparen token: "(". Instead, I was expecting to see one of the following:

        - A "word" token
```

**Comment by oguimbal** [comment]
Hi,

Sorry for the delay, I dont have enough time to work on this these days.

I'll try to have a look when I have time.

Thanks for the report, and for the kind words

**Comment by oguimbal** [comment]
Hi ! Fixed in `pgsql-ast-parser@9.1.0`. Sorry for the delay :)
